tests/console/journal_check.pm

```diff
@@ -3,7 +3,9 @@
 # Copyright 2017-2021 SUSE LLC
 # SPDX-License-Identifier: FSFAP
 
-# Summary: Basic journal tests
+# Summary: Check system journal for errors given a list of known patterns
+#          referring to known bugs. This module will fail in case an unknown
+#          message was found.
```

Comment:
I would add, that it also fails in case a failing systemd service is detected (see Lines 97 and following)

Reply:
You are right. Thanks for pointing that out. I have added it in the newly pushed version.